markdown/tuto-040-A-ping.md

```diff
@@ -36,9 +36,10 @@ in each tutorial.
 Before we can run these experiments however, we need to have
 
 * a valid lease set in the booking system
-* the 2 nodes `fit01` and `fit02` up and running
+* the 2 nodes `fit01` and `fit02` up and running with the `baleine` disk image
+* the `faraday.repo/tutorial` Docker image running on the fit nodes
 
-For this first tutorial we will assume that these 2 steps have been
+For this first tutorial we will assume that these 3 steps have been
```

Comment:
if it ain't broke don't fix it; 2 is good enough

Reply:
I mean, we now have 3 steps don't we ?